go/vt/sqlparser/ast_format.go

```diff
@@ -1199,6 +1199,9 @@ func (node *Subquery) Format(buf *TrackedBuffer) {
 
 // Format formats the node.
 func (node *DerivedTable) Format(buf *TrackedBuffer) {
+	if node.LateralPresent {
+		buf.WriteString("lateral")
```

Comment:
We generally have a space between a keyword and opening parenthesis . Could you add a space here too

Reply:
Done